### ISSUE TITLE
Drop downs missing ARIA roles

### ISSUE THREAD

**Issue by coleweusthem** [asc-csa/AlouetteApp#19]
Certain ARIA roles must be contained by particular elements

<img width="580" alt="Screen Shot 2021-03-10 at 1 56 22 PM" src="https://user-images.githubusercontent.com/46227765/110674576-67bf2100-81a8-11eb-983c-217ab874e270.png">

**Comment by coleweusthem** [comment]
no longer an issue
